doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
They can be ordered without dependency, aren't they? Dependency enforce order, the opposite is not true.